restaurants/models.py

```diff
@@ -79,11 +79,25 @@ class Review(models.Model):
     restaurant = models.ForeignKey(Restaurant, related_name='reviews', on_delete=models.CASCADE)
     rating = models.DecimalField(max_digits=3, decimal_places=1)
     comment = models.TextField()
-    visited = models.BooleanField(default=False)
+
+    def __str__(self):
+        return f'{self.user.username} - {self.restaurant.title}'
+    
+class Bookmark(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    restaurant = models.ForeignKey(Restaurant, related_name='bookmarks', on_delete=models.CASCADE)
     bookmarked = models.BooleanField(default=False)
```

Comment:
Why is bookmarked field needed ? We will create Bookmark model object when user bookmarks a restaurant and delete it when they remove the bookmark.